SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -401,14 +394,27 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	// If we're the client, we can now ask the server to spawn our controller.
-	if (!IsServer())
+	if (IsServer())
+	{
+		if (GlobalStateManager && !ServerConnection)
+		{
+			GlobalStateManager->SetCanBeginPlay(true);
+			GlobalStateManager->TriggerBeginPlay();
+			GlobalStateManager->SetAcceptingPlayers(true);
+		}
```

Comment:
Is this an error if either of the conditions fail? If so we should print and abort.

Reply:
Yes. We start listening to PostLoadMapWithWorld when the net driver is created. A map is loaded before we have actually created the GlobalStateManager.  Added logging for this and removing the serverconnection check as it seems to check if the worker is a server or not (which we already do).

Reply:
GlobalStateManager can be null as The OnMapLoaded function is called before we create GlobalStateManager.

Reply:
This is an error then?